<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>baza</class>
 <widget class="QDialog" name="baza">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>531</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="UPDATE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE TABLE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Brisanje_Redova">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UNIJETI ZELJENI ID ZA BRISANJE REDA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="izmjeni_tabelu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Izmjeni</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unesiUtabelu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Unesi u tabelu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="obrisi_tabelu">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Obrisi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UNIJETI ZELJENI ID ZA IZMJENU REDA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_izmjena_redova">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="izaberi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Izaberi</string>
   </property>
  </widget>
  <widget class="QLabel" name="stanje_za_izmjenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>186</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Broj dresa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Ime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Prezime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Poeni</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Korisnicko ime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Sifra</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="broj_dresa"/>
      </item>
      <item>
       <widget class="QLineEdit" name="ime"/>
      </item>
      <item>
       <widget class="QLineEdit" name="prezime"/>
      </item>
      <item>
       <widget class="QLineEdit" name="poeni"/>
      </item>
      <item>
       <widget class="QLineEdit" name="korisnicko_ime"/>
      </item>
      <item>
       <widget class="QLineEdit" name="sifra"/>
      </item>
      <item>
       <widget class="QLineEdit" name="ID"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_OBRISI">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
